javascript/index.js

```diff
@@ -1 +1,114 @@
-console.log('radi')
\ No newline at end of file
+const colors = ["white", "red", "blue", "green", "pink", "yellow"];
+
+console.log('colors / for loop:');
+for (let i = 0; i < colors.length; i ++){
+    
+    console.log(colors[i]);
+}
+
+console.log('colors / foreach / index:');
+for (let index in colors) {
+
+    console.log(index + ': ' + colors[index]);
+}
+
+console.log('colors / foreach / element:');
+for (let color of colors) {
+
+    console.log(color);
+}
+
+console.log('colors / array.forEach / element:');
+colors.forEach((color) => {
+
+    console.log(color);
+});
+
+console.log('rotate array:')
+function rotateArrayToRight (baseArray, steps) {
+
+    for (let i = 0; i < steps; i++) {
+
+        baseArray.unshift(baseArray.pop());
+    }
+
+    return baseArray;
+
+}
+console.log('original array: ' + colors);
+const rotatedArray = rotateArrayToRight(colors, 2);
+console.log('rotated array: ' + rotatedArray);
+
+
+function reduceArrayFromIndex (array, index) {
+
+    const summ = array.slice(index).reduce((summ, currentValue) => {
+        return summ + currentValue;
+    }, 0);
+    console.log('summ: ' + summ);
+
+}
```

Comment:
Write a function that uses the native Array .reduce method to sum up an array of numbers, but starting at 50.

znaci da krene da sabira od broja 50, ne da krene od vrednosti koja se nalazi na indeksu 50.

```
(array)=> array.reduce((acc, element)=> acc + element, 50)
```